AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx

```diff
@@ -1,8 +1,18 @@
 import * as React from 'react';
 import {Panel} from 'react-bootstrap';
 import moment from 'moment-timezone';
+// import { render } from "react-dom";
+// import { renderToString } from "react-dom/server";
```

Comment:
Do we need these imports? can we remove the comments?

Reply:
These have now been deleted and committed